### DISCUSSION TITLE
What is this "type-or-unit parameter" mentioned in the counter() doc?

### DISCUSSION THREAD

**Discussion by adrienp** [mdn/mdn-community#622] [Content] [answered]
Hi! I'm curious about this `type-or-unit` parameter that's mentioned in the `counter()` doc.

https://github.com/mdn/content/blob/e89a324e54cfc261f00ae08052d63c697cd99ba2/files/en-us/web/css/counter()/index.md?plain=1#L26

It makes it sound like there's another parameter like `attr()`'s `<type-or-unit>`, but there's no documentation of that parameter for `counter()`. Not seeing any reference to this parameter in the spec either: https://www.w3.org/TR/css-lists-3/#funcdef-counter

Am I missing something? Or should this reference to `type-or-unit` here be removed?

Thanks!

**Comment by wbamberg** [comment, marked as answer]
I can't find any reference to it either. Perhaps it's a copy/paste error from https://developer.mozilla.org/en-US/docs/Web/CSS/attr() ?

**Reply by hamishwillee** [reply]
Sounds likely - I just did some work in this area and there is definitely no mention of that argument. Nor is the bit about it being experimental outside of use in `content` strictly true - the spec shows it used there but indicates that it returns a string, and hence can be used anywhere a string can be used. 

I have updated the docs in https://github.com/mdn/content/pull/12562.

**Comment by pepelsbey** [comment]
Closing as answered/settled.
